internal/oidc/client.go

```diff
@@ -73,3 +125,64 @@ func (c InternalClient) IsPublic() bool {
 func (c InternalClient) GetAudience() fosite.Arguments {
 	return c.Audience
 }
+
+// GetResponseModes returns the valid response modes for this client.
+//
+// Implements part of fosite.OpenIDConnectClient.
+func (c InternalClient) GetResponseModes() []fosite.ResponseModeType {
+	return c.ResponseModes
+}
+
+// GetRequestURIs returns the valid request URIs for this client.
+//
+// Implements part of fosite.OpenIDConnectClient.
+func (c InternalClient) GetRequestURIs() []string {
+	return c.RequestURIs
+}
+
+// GetJSONWebKeys returns the jose.JSONWebKeySet for this client.
+//
+// This should only be set for RP's that don't understand the JSONWebKeysURI, and should not be set along side this.
+// Implements part of fosite.OpenIDConnectClient.
+func (c InternalClient) GetJSONWebKeys() *jose.JSONWebKeySet {
+	return c.JSONWebKeys
+}
+
+// GetJSONWebKeysURI returns the URI containing the JWKS for this client.
+//
+// This should generally always be set unless the client does not understand the JWKS URI in which case
+// GetJSONWebKeys should be set instead.
+//
+// Implements part of fosite.OpenIDConnectClient.
+func (c InternalClient) GetJSONWebKeysURI() string {
+	return c.JSONWebKeysURI
+}
+
+// GetRequestObjectSigningAlgorithm returns the object sig alg.
+//
+// Implements part of fosite.OpenIDConnectClient.
+func (c InternalClient) GetRequestObjectSigningAlgorithm() string {
```

Comment:
should be in a dedicated PR as discussed.

Reply:
Removed.